spec/plugins/data_table/general.js

```diff
@@ -557,13 +557,57 @@ describe("DataTable Plugin", function() {
       expect(reorderableSpy).not.toHaveBeenCalled();
     });
 
+    it("should default column resize to off", function() {
+      app.view.dataTable.row("abc", {
+        columns : [
+          { attr: "name", title: "Name" },
+          { attr: "age", title: "Age" }
+        ]
+      });
+
+      app.view.dataTable("def", {
+        rowClassName : "abc"
+      });
+
+      var table = new app.Views.def({ collection : collection }).render();
+
+      expect(table.resizableColumns).toEqual(false, "Table resizableColumns default");
+      expect(table._colReorder.s.allowResize).toEqual(false, "ColReorder allowResize");
+      expect(table._colReorder.s.allowReorder).toEqual(true, "ColReorder allowReorder");
+    });
+
+    it("should allow columns to be resized with flag", function() {
+      app.view.dataTable.row("abc", {
+        columns : [
+          { attr: "name", title: "Name" },
+          { attr: "age", title: "Age" }
+        ]
+      });
+
+      app.view.dataTable("def", {
+        rowClassName : "abc",
+        resizableColumns : true
+      });
+
+      var table = new app.Views.def({ collection : collection }).render();
+
+      expect(table.resizableColumns).toEqual(true, "Table resizableColumns default");
+      expect(table._colReorder.s.allowResize).toEqual(true, "ColReorder allowResize");
+      expect(table._colReorder.s.allowReorder).toEqual(true, "ColReorder allowReorder");
+
+      // these should set to off as they cause display issues
+      expect(table._colReorder.s.bAddFixed).toEqual(false, "ColReorder bAddFixed");
+      expect(table._colReorder.s.bResizeTableWrapper).toEqual(false, "ColReorder bResizeTableWrapper");
```

Comment:
those are from long ago (Oct 8), and we discussed it back then. I thought you had turned off line length altogether based on that.  I haven't seen a hound error about line length since (but we haven't done much dev here since either)